installation/scripts/installer.sh

```diff
@@ -91,9 +91,9 @@ if [ ${ADMIN_PASSWORD} ]; then
     COMBO_YAML=$(sed 's/global\.adminPassword: .*/global.adminPassword: '"${ADMIN_PASSWORD}"'/g' <<<"$COMBO_YAML")
 fi
 
-if [ $LOCAL ]; then
+if [ ${LOCAL} ]; then
     MINIKUBE_IP=$(minikube ip)
-    COMBO_YAML=$(sed 's/test\.acceptance\.ui\.minikubeIP: .*/test\.acceptance\.ui\.minikubeIP: '"${MINIKUBE_IP}"'/g' <<<"$COMBO_YAML")
+    COMBO_YAML=$(sed 's/minikubeIP: .*/minikubeIP: '"${MINIKUBE_IP}"'/g' <<<"$COMBO_YAML")
```

Comment:
I'd add a dot just in case to make sure it is replacing only `minikubeIP:` and not `thisisnotminikubeIP:`  
```suggestion
    COMBO_YAML=$(sed 's/\.minikubeIP: .*/.minikubeIP: '"${MINIKUBE_IP}"'/g' <<<"$COMBO_YAML")
```